code/controllers/subsystem/tts.dm

```diff
@@ -183,6 +183,9 @@ SUBSYSTEM_DEF(tts)
 		var/identifier = current_request.identifier
 		if(current_request.requests_errored())
 			current_request.timed_out = TRUE
+			var/datum/http_response/normal_response = current_request.request.into_response()
+			var/datum/http_response/blips_response = current_request.request_blips.into_response()
+			log_tts("TTS HTTP request errored | Normal: [normal_response.error] | Blips: [blips_response.error]")
```

Comment:
if `normal_response` and `blips_response` are lists, pass them as data.
If they are json, turn them into lists and pass them as data.

Reply:
they are just strings of text